docs/source/commands/dev.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: The Rover dev command
+description: Query a locally federated GraphQL API
+---
+
+> ⚠️ **Note:** The `rover dev` command is _only_ for local development. Do **not** run this command in production.
+
+A **subgraph** is a graph that contributes to the composition of a federated **supergraph**:
+
+```mermaid
+graph BT;
+  gateway(["Supergraph (A + B + C)"]);
+  serviceA[Subgraph A];
+  serviceB[Subgraph B];
+  serviceC[Subgraph C];
+  gateway --- serviceA & serviceB & serviceC;
+```
+
+You can use Rover to start a local router that can query across one or more running GraphQL APIs (i.e., subgraphs)
+through one endpoint (i.e., supergraph). As you add, edit, and remove subgraphs, Rover automatically
+composes their separate schemas into a unified supergraph schema that the router reloads and serves. 
+
+Think plug-n-play USB devices but with your GraphQL APIs!
+
+## Creating a federated session
+
+### Starting the session
+
+Before using Rover's `dev` command, you first need to have access to a running GraphQL API (subgraph). You'll need the URL the GraphQL API is serving and the location of the schema in the filesystem. If you don't have a local schema file, your subgraph must enable introspection.
+
+An example `rover dev` command for running a local subgraph might look something like this:
+
+`rover dev --name products --schema ./products.graphql --url http://localhost:4000`
+
+The above command starts the `rover dev` session, watching `./products.graphql` for changes to the schema, and starts up a local Apollo Router at the default endpoint (i.e., `http://localhost:3000`). This session also listens for other `rover dev` processes that include different subgraph schemas.
+
+### Stopping the session
+
+If you stop the first `rover dev` session (by pressing `CTRL+C`), it will shut down the router, and any attached `rover dev` processes are detached and shut down.
```

Comment:
Sorry to keep adding stuff! How do you feel about this slight tweak:
```suggestion
If you stop the first `rover dev` process (by pressing `CTRL+C`), it will shut down the router, and any attached `rover dev` processes are detached and shut down as the session ends.
```